docs/administration/csv-mappers.md

```diff
@@ -0,0 +1,72 @@
+# CSV Mappers
+
+CSV Mappers allow to parse CSV files in a given, specific format. The mappers are created and configured by administrators and
+then made available to users who import CSV files, for instance inside a report or in global import view, and want to extract informations inside these files.
+
```

Comment:
We have to detail here which capability allows a user to use the feature. "Administrator" is too broad

Reply:
addressed by latest changes
> The mappers are created and configured by users with Manage CSV mappers capability and
then made available to users who import CSV files